### PR TITLE
Feat/po_project_members

### PR DESCRIPTION
**Pull request by andreas-unleash**
- Initial separation on project members into it's own component
- Styling fixes
- Hide new ToggleTypesWidget behind a flag

Members widget: https://linear.app/unleash/issue/1-586/project-overview-members-widget
## About the changes

Closes #

### CODE REVIEW

File: frontend/src/component/project/Project/ProjectInfo/ProjectInfo.styles.ts [hunk 1]
```diff
@@ -53,11 +53,11 @@ export const StyledParagraphSubtitle = styled('p')(({ theme }) => ({
 
 export const StyledParagraphGridRow = styled('div')(({ theme }) => ({
     display: 'grid',
-    gridGap: theme.spacing(2),
-    gridTemplateColumns: '10% 70% 20%',
-    margin: theme.spacing(1, 2, 1, 2),
+    gridGap: theme.spacing(1),
+    gridTemplateColumns: '20px auto auto',
```
Comment (Tymek) [inline frontend/src/component/project/Project/ProjectInfo/ProjectInfo.styles.ts:57]: `20px` can be expressed using `theme.spacing`

File: frontend/src/component/project/Project/ProjectInfo/ProjectInfo.styles.ts [hunk 2]
```diff
@@ -72,8 +72,9 @@ export const StyledParagraphEmphasizedText = styled('p')(({ theme }) => ({
 }));
 
 export const StyledCount = styled('p')(({ theme }) => ({
-    fontSize: '1.2rem',
+    fontSize: '1.25rem',
```
Comment (Tymek) [inline frontend/src/component/project/Project/ProjectInfo/ProjectInfo.styles.ts:75]: `theme.typography.h2.fontSize`?

File: frontend/src/component/project/Project/ProjectInfo/ProjectInfo.tsx [hunk 3]
```diff
@@ -1,19 +1,12 @@
...
-    StyledParagraphEmphasizedText,
-    StyledParagraphSubtitle,
-    StyledSpanLinkText,
-} from './ProjectInfo.styles';
+import { StyledDivContainer } from './ProjectInfo.styles';
 import { IFeatureToggleListItem } from '../../../../interfaces/featureToggle';
 import { HealthWidget } from './HealthWidget';
 import { ToggleTypesWidget } from './ToggleTypesWidget';
+import { ProjectMembersWidget } from './ProjectMembersWidget';
+import useUiConfig from '../../../../hooks/api/getters/useUiConfig/useUiConfig';
```
Comment (Tymek) [inline frontend/src/component/project/Project/ProjectInfo/ProjectInfo.tsx:9]: import cleanup

File: frontend/src/component/project/Project/ProjectInfo/ProjectMembersWidget.tsx [hunk 4]
```diff
@@ -0,0 +1,42 @@
+import {
+    StyledArrowIcon,
+    StyledDivInfoContainer,
+    StyledLink,
+    StyledParagraphEmphasizedText,
+    StyledParagraphSubtitle,
+    StyledSpanLinkText,
+} from './ProjectInfo.styles';
+import useUiConfig from '../../../../hooks/api/getters/useUiConfig/useUiConfig';
```
Comment (Tymek) [inline frontend/src/component/project/Project/ProjectInfo/ProjectMembersWidget.tsx:9]: use import alias

File: frontend/src/component/project/Project/ProjectOverview.tsx [hunk 5]
```diff
@@ -30,7 +30,6 @@ const StyledContentContainer = styled(Box)(() => ({
     display: 'flex',
     flexDirection: 'column',
     width: '100%',
-    minWidth: 0,
```
Comment (Tymek) [inline frontend/src/component/project/Project/ProjectOverview.tsx:33]: Why does this revert my bugfix?

Comment (andreas-unleash) [reply frontend/src/component/project/Project/ProjectOverview.tsx:33]: Not intentional

Comment (andreas-unleash) [reply frontend/src/component/project/Project/ProjectOverview.tsx]: Reverted

**Review by Tymek — APPROVED** [review]

**Pull request merged**